R/app-server.R

```diff
@@ -283,27 +293,50 @@ app_server <- function(input, output, session) {
     })
 
     ## Successes box
-    output$successes <- renderUI({
+    observe({
       successes <- purrr::map_lgl(res(), function(x) {
         inherits(x, "check_pass")
       })
-      report_results(res()[successes], emoji_prefix = "check")
+      output$successes <- renderUI({
+        report_results(res()[successes], emoji_prefix = "check")
+      })
+      reporting_titles$success <- paste0("Successes (",
+                                         as.character(sum(successes)),
+                                         ")")
```

Comment:
I've been using `glue()` in new code to try to make the construction of these messages a little more concise. The syntax here would be: `glue::glue("Successes ({sum(successes)})")` (`as.character()` isn't needed). Up to you whether you want to use this or not, just an idea, but either way will you please update the style of these lines? `"Successes (",` should go on a new line.

Reply:
I think the `glue` option is better. I'll change it to that. It should make the line short enough to not need to drop down to a new line.